common/structures/conditions.py

```diff
@@ -176,20 +176,14 @@ def __init__(self,
         self.wind_direction = wind_direction
         self.time_blocks = time_blocks
 
-    # TODO ERROR: This method needs to be fixed: speed is not defined unless the if condition holds.
     def get_wind_conditions(self, azimuth) -> np.ndarray:
-        if np.asarray(self.wind_speed).ndim == 0:
-            speed = np.full(len(azimuth), self.wind_speed.to(u.m / u.s).value) * u.m / u.s
-        
+     
```

Comment:
I'm not totally clear now on what will happen if the `ndim` of `self.wind_speed` is `0`, or can this not happen?